docs/content/01_introduction/getting-started.md

```diff
@@ -27,6 +27,8 @@ yarn
 yarn start
 ```
 
+If you get an error `Incorrect integrity when fetching from the cache` when creating your app, try running `yarn cache clean` first.
```

Comment:
Do we know why this is happening? I'm not sure I like adding this to the getting started section. I think the installation section would be better.